src/test/scala/com/mongodb/spark/rdd/partitioner/MongoPaginationPartitionerSpec.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2016 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.spark.rdd.partitioner
+
+import org.bson.{BsonDocument, BsonMaxKey, BsonMinKey, Document}
+import com.mongodb.spark.RequiresMongoDB
+
+class MongoPaginationPartitionerSpec extends RequiresMongoDB {
+
+  // scalastyle:off magic.number
+  "MongoPaginationPartitioner" should "partition the database as expected" in {
+    loadSampleData(10)
+
+    val partitions = MongoPaginationPartitioner.partitions(mongoConnector, readConfig.copy(partitionSizeMB = 1))
+    partitions.length should equal(11)
+    partitions.head.locations should not be empty
```

Comment:
After looking at similar assertions for the sample-based partitioner, it occurred to me that perhaps it's because we can't reliably determine the partition points,